StratisCore.UI/src/app/setup/create/create.component.css

```diff
@@ -0,0 +1,8 @@
+/* Allows for vertical scrolling if form gets too long. */
+.createWallet {
+  overflow-y: scroll;
+  max-height: 100vh;
+  display: block !important;
+  padding-top: 25px;
+  padding-bottom: 25px;
+}
```

Comment:
Probably best to move this to _custom.scss, we're not using the component CSS anywhere.

Reply:
Sure, I'll move this